CHANGELOG.asciidoc

```diff
@@ -2,6 +2,198 @@
 // each other on Github, so don't worry too much on using the right prefix.
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
+
+[[release-notes-8.16.3]]
+=== Beats version 8.16.3
+https://github.com/elastic/beats/compare/v8.16.2\...v8.16.3[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Fix FQDN being lowercased when used as `host.hostname` {issue}39993[39993]
```

Comment:
This (and maybe other) entries were already released. See https://github.com/elastic/beats/blob/e3b56a20aff3e0492f25c42e1c7bff00ecc6c653/CHANGELOG.asciidoc#L451

I suspect the whatever base the changelog process is using to move the entries around is not up to date, either that or merge conflicts did this.